src/docfx/docset/Docset.cs

```diff
@@ -33,6 +33,12 @@ internal class Docset
         /// </summary>
         public RestoreMap RestoreMap { get; }
 
+        /// <summary>
+        /// Gets the git repository this docset belongs to. Null if there is no git repo.
+        /// Multiple repositories inside a single docset is not supported.
```

Comment:
>Multiple repositories inside a single docset is not supported. [](start = 12, length = 62)

Some community users may want this feature. They may use submodule or CRR(in v3) to organize the docset.